src/backend/core/models.py

```diff
@@ -154,9 +154,9 @@ class UserManager(auth_models.UserManager):
     Override user manager to get the related contact in the same query by default (Contact model)
     """
 
-    def get_queryset(self):
+    def get_queryset(self):  # pylint: disable=useless-parent-delegation
         """Always select the related contact when doing a query on users."""
-        return super().get_queryset().select_related("profile_contact")
+        return super().get_queryset()
```

Comment:
This custom UserManager has no reason to remain. Can I discard it, @sampaccoud ? We can re-add it if needed if desk and contact "merge"  again

Reply:
ok! why don't you remove the whole function override as it is now useless?